helm/vitess/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.0.0-1 - 2020-04-16
```

Comment:
To date, I've used the X.Y.Z versioning to handle chart changes like this, and the `-N` to mark updated Docker images / binaries that work with the same version. I think we should also add the proxy flag introduced in #6058 / #6062 and call this a 2.0.1 release. I can tag the new images.
```suggestion
## 2.0.1-0 - 2020-04-16
```